widgets/card_widget.php

```diff
@@ -10,35 +10,35 @@
  *
  * @since 1.0.0
  */
-class Wasyo_Image_Card extends \Elementor\Widget_Base
+class Image_Card extends \Elementor\Widget_Base
 {
 
 	/**
 	 * Get widget name.
 	 *
-	 * Retrieve wasyo card widget name.
+	 * Retrieve Image card widget name.
 	 *
 	 * @since 1.0.0
 	 * @access public
 	 * @return string Widget name.
 	 */
 	public function get_name()
 	{
-		return 'wasyo_card';
+		return 'Image_card';
 	}
 
 	/**
 	 * Get widget title.
 	 *
-	 * Retrieve Wasyo-Card widget title.
+	 * Retrieve Image-Card widget title.
```

Comment:
For your benefit, i'm pointing out that you have inconsistent naming! Where did the dash (-) in the name come from? The widget is called CardWidget, no  dashes.